p2panda-rs/Cargo.toml

```diff
@@ -44,7 +44,7 @@ serde_cbor = "0.11.2"
 serde_repr = "0.1.7"
 serde_json = "1.0.59"
 thiserror = "1.0.29"
-yamf-hash = "0.1.2"
+yasmf-hash = "0.1.0"
```

Comment:
Should `yasmf-hash` version numbers track `yamf-hash` version numbers?

Reply:
I can imagine `yasmf-hash` might add hashes in a different rhythm than `yamf-hash`, so maybe not?